src/lib/model/OutboundTransfersModel.js

```diff
@@ -738,7 +740,7 @@ class OutboundTransfersModel {
                     // next transition is to resolvePayee
                     await this.stateMachine.resolvePayee();
                     this._logger.log(`Payee resolved for transfer ${this.data.transferId}`);
-                    if(this.stateMachine.state === 'payeeResolved' && !this._autoAcceptParty) {
+                    if (this.stateMachine.state === 'payeeResolved' && !this._autoAcceptParty) {
```

Comment:
please dont change whitespace unless there is a very good reason to do so.

Reply:
Deleted